### PR TITLE
LSR-717 Android example of creating a custom interactor against delicious API

### PR DESCRIPTION
**Pull request by nhpatt**

### CODE REVIEW

File: android/samples/addbookmarkscreenlet/src/main/java/com/liferay/mobile/screens/bookmark/interactor/AddDeliciousInteractorImpl.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+					OkHttpClient client = new OkHttpClient();
+
+					Request add = new Request.Builder()
+						.url("https://api.del.icio.us/api/v1/posts/add?url=" + url + "&description=" + title)
+						.headers(headers)
+						.build();
+
+					client.newCall(add).execute();
+
```
Comment (jmnavarro) [inline android/samples/addbookmarkscreenlet/src/main/java/com/liferay/mobile/screens/bookmark/interactor/AddDeliciousInteractorImpl.java]: Add comment: why send a second request?

File: android/samples/addbookmarkscreenlet/src/main/java/com/liferay/mobile/screens/bookmark/interactor/AddDeliciousInteractorImpl.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+		super(targetScreenletId);
+	}
+
+	public void addBookmark(final String url, final String title, long folderId) throws Exception {
+		new Thread(new Runnable() {
+			@Override
+			public void run() {
+				try {
+
+					Headers headers = Headers.of("Authorization", "Bearer 11336429-c2378f3c44a29f593e31aa4f5521e4dd");
```
Comment (jmnavarro) [inline android/samples/addbookmarkscreenlet/src/main/java/com/liferay/mobile/screens/bookmark/interactor/AddDeliciousInteractorImpl.java]: Add comment: how to generate this token? What if this token is invalidated?

**Comment by nhpatt** [comment]
New comments :)

**Comment by jmnavarro** [comment]
:+1:

**Pull request merged**